pkg/client/compose/service_test.go

```diff
@@ -6,8 +6,9 @@ import (
 	"slices"
 	"strings"
 	"testing"
-
+	
```

Comment:
I added [`.editorconfig` ](https://editorconfig.org/) to the repo earlier today; hope your IDE supports because it'll help to get rid of these trailing whitespaces and also ensure all files have trailing new lines :raised_hands: